axelrod/strategies/grudger.py

```diff
@@ -171,3 +171,63 @@ def reset(self):
         Player.reset(self)
         self.grudged = False
         self.grudge_memory = 0
+
+
+
+class GrudgerAlternator(Player):
+    """
+    A player starts by cooperating until the first opponents defection,
+    then alternates D-C.
+
+    Names:
+
+    - c_then_per_dc: [PRISON1998]_
+    - GrudgerAlternator
```

Comment:
Can you make this something like:

```
- Grudger Alternator: Original name by Geraint Palmer
```

Or whatever self attribution you would like to use?